.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This is recommendations of extensions to add when setting up the repo, ( maybe would check for more like react snippets etc. )